xilem/src/view/text_input.rs

```diff
@@ -70,6 +73,30 @@ impl<State: 'static, Action: 'static> TextInput<State, Action> {
         self
     }
 
+    /// Set the insertion caret's color.
+    ///
+    /// This overwrites the default `CaretColor` property for the inner `TextArea` widget.
+    pub fn caret_color(self, color: Color) -> Prop<CaretColor, Self, State, Action> {
+        self.prop(CaretColor { color })
+    }
+
+    /// Set the selection's color.
+    ///
+    /// This overwrites the default `SelectionColor` property for the inner `TextArea` widget.
+    pub fn selection_color(self, color: Color) -> Prop<SelectionColor, Self, State, Action> {
+        self.prop(SelectionColor { color })
+    }
+
+    /// Set the selection's color when the window is unfocused.
+    ///
+    /// This overwrites the default `UnfocusedSelectionColor` property for the inner `TextArea` widget.
+    pub fn unfocused_selection_color(
+        self,
+        color: Color,
+    ) -> Prop<UnfocusedSelectionColor, Self, State, Action> {
+        self.prop(UnfocusedSelectionColor(SelectionColor { color }))
+    }
+
```

Comment:
I *think* these also should be made available on `Prose`. However, it's possible that I'm missing some detail which makes that worth skipping for now.